packages/client/package.json

```
@@ -10,13 +10,15 @@
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.4",
     "babel-loader": "8.1.0",
+    "formik": "^2.2.9",
```

We don't seem to be using formik anywhere, let's get rid of it.
```suggestion
    "formik": "^2.2.9",
```

---

Thank you, sorry we missed that!